pubspec.yaml

```diff
@@ -40,5 +51,20 @@ flutter:
     - assets/activities/
     - assets/speakers/
     - assets/sponsors/
+    - assets/lotties/
+    - assets/rive/
+    - assets/audio/
     - shorebird.yaml
   uses-material-design: true
+
+  fonts:
+    - family: Poppins
+      fonts:
+        - asset: assets/fonts/poppins/Poppins-Black.ttf
+        - asset: assets/fonts/poppins/Poppins-Bold.ttf
+        - asset: assets/fonts/poppins/Poppins-ExtraBold.ttf
+        - asset: assets/fonts/poppins/Poppins-Light.ttf
+        - asset: assets/fonts/poppins/Poppins-Medium.ttf
+        - asset: assets/fonts/poppins/Poppins-Regular.ttf
+        - asset: assets/fonts/poppins/Poppins-SemiBold.ttf
+        - asset: assets/fonts/poppins/Poppins-Thin.ttf
```

Comment:
Are all these needed? They will increase the overall bundle size and I prefer to keep the app as light as possible.

Reply:
I will remove the font files.